src/main/resources/application.properties

```diff
@@ -9,6 +9,4 @@ spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 #jpa configuration
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
-
-
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
```

Comment:
In all config files, have an extra line at the end.

Reply:
done